index.md

```diff
@@ -118,15 +123,15 @@ Often wireless lapel microphones are best so that speakers can move around. When
 
 Have good lighting on their face and upper body. Avoid distracting backgrounds, such as bright sunlight or flashing light.
 
-### Arrange for [ALDs/hearing loops](#ald){:.termref}, [interpreters](#terps){:.termref}, and/or [CART](#cart){:.termref} writers, as needed
+### Arrange for assistive listening devices (ALD)s, interpreters, and captioners, as needed
 
-*(Assistive listening devices ([ALDs](#ald){:.termref}), hearing loops, or induction loops; and Computer Aided Real–Time Captioning or Communication Access Realtime Translation ([CART](#cart){:.termref}) are explained in the [Terminology](#terms) section.)* 
+*(Assistive listening devices ([ALDs](#ald){:.termref}), [interpreters](#terps){:.termref}, and ([captioners](#captions){:.termref}) are explained in the [Terminology](#terms) section.)* 
 
-Work with participants and service providers to ensure that important details are taken care of. For example, power and a screen are provided for CART. Or microphones and the speakers' computer sound output is connected to the [ALD/hearing loop](#ald){:.termref} system.
+Work with participants and service providers to ensure that important details are taken care of. For example, electricity and a screen are provided for the captioning. Or microphones and the presenter’s computer sound output is connected to the ALD system.
```

Comment:
Not sure electricity is the key concern... ensuring the captioner (who are often remote) can hear the audio clearly is usually the biggest concern

Reply:
I've heard of in-person captioner needing power/electricity and not having it. :)

Changed to:
> For example, a remote captioner can hear the audio clearly, an in-person captioner has the connections they need, and a screen is provided for participants to read the captions.

Reply:
> I've heard of in-person captioner needing power/electricity and not having it. :)

Wow!?!